keystone_api/apps/allocations/tasks.py

```diff
@@ -59,17 +59,19 @@ def update_limit_for_account(account_name: str, cluster: Cluster) -> None:
 
     # Filter account's allocations to those that are active, and determine their total service unit contribution
     active_sus = acct_alloc_query.filter(request__active__lte=date.today(), request__expire__gt=date.today()) \
-                                 .aggregate(Sum("awarded"))
+                                 .aggregate(Sum("awarded"))['awarded__sum'] or 0
```

Comment:
Had a couple of these where I needed to index into the dictionary object that comes out of aggregating.